contracts/staking/contracts/src/interfaces/IStakingProxy.sol

```diff
@@ -17,9 +17,12 @@
 */
 
 pragma solidity ^0.5.9;
+pragma experimental ABIEncoderV2;
 
+import "./IStructs.sol";
 
-interface IStakingProxy /* is IStaking */
+
+contract IStakingProxy /* is IStaking */
```

Comment:
Now that this is a contract, we could actually inherit `IStaking` here. Is there a reason not to?

Reply:
Actually we cannot, or the StakingProxy doesn't deploy (unimplemented functions).